feelslikemonday/app/src/main/java/com/example/feelslikemonday/model/MoodEvent.java

```diff
@@ -0,0 +1,102 @@
+package com.example.feelslikemonday.model;
+
+import android.location.Location;
+
+//Todo: Do we give user's the ability to change time and date?
+// Also, should we have a list of all options stored as a static attribute?
+//The date and time are made as strings since it should be easier to perform operations to get them.
+//It doesn't seem we perform any operations that doesn't warrant them not to be strings
+public class MoodEvent {
+    private String date;
+    private String time;
+    private String emotionalState;
+    private String reason;
+    private String socialSituation;
+    private MoodType moodType;
+    private byte[] image;
+
+    //To be attached when editing
+    private Location location;
+
+    //Only one constructor since it's not possible to have overloading with different constructors
+    public MoodEvent(String date, String time, String emotionalState, String reason, MoodType moodType, String socialSituation) {
```

Comment:
wait explain this? why only one constructor? so then doe this mean if they do not add a photo/location/socialsituation/reason they will be set as null?

Reply:
Old comment. Good catch